src/popup/utils/utils.ts

```diff
@@ -443,3 +443,11 @@ export function getAeFee(value: number | string) {
 export function openInNewWindow(url: string) {
   window.open(url, '_blank');
 }
+
+export function prepareConsensusLabel(
+  confirmedBy: number,
+  confirmationsRequired: number,
+  totalConfirmations: number,
+) {
+  return `${confirmedBy}/${confirmationsRequired} ${i18n.t('common.of')} ${totalConfirmations}`;
+}
```

Comment:
In my opinion this should be a component.

Reply:
I'll create a task for it with the refactor (which I tried to avoid in this PR) of all places where it exists